### DISCUSSION TITLE
UI parameter interpolation question

### DISCUSSION THREAD

**Discussion by yriveiro** [argoproj/argo-workflows#9495] [Q&A]
Hi,

I'm launching a DAG and I notice that the UI doesn't resolve the `workflow.parameters`, is this by design of is a bug?

<img width="630" alt="Screenshot 2022-09-02 at 16 39 41" src="https://user-images.githubusercontent.com/934763/188190298-216b1203-2d05-456d-9eda-83afbff9fbe9.png">

**Comment by leveryd** [comment]
i think it is a design.
